app/src/main/java/de/danoeh/antennapod/dialog/EpisodesApplyActionFragment.java

```diff
@@ -386,8 +386,8 @@ private void refreshCheckboxes() {
             mListView.setItemChecked(i, checked);
         }
         refreshToolbarState();
-        toolbar.setTitle(getResources().getQuantityString(R.plurals.num_selected_label,
-                checkedIds.size(), checkedIds.size()));
+//        toolbar.setTitle(getResources().getQuantityString(R.plurals.num_selected_label,
+//                checkedIds.size(), checkedIds.size()));
```

Comment:
I think the `EpisodesApplyActionFragment` is unused now. So please delete it :)

Reply:
> * In the queue, the multi-select context menu item is not the top-most one. I think it should consistently be either the first or the last one on all screens.

Ok, i'll fix this

> * In the queue, you can still drag around episodes. We only store the ID of the selected items (not the position) but it still feels a bit risky that episodes can be dragged in select mode.

Yes, noticed this too, i just hadn't gotten to it. I'll fix this too.